src/build/tenscript/final_phase.rs

```diff
@@ -26,6 +27,7 @@ pub struct FinalPhase {
     pub muscle_movement: Option<MuscleMovement>,
     pub pretense_factor: Option<f32>,
     pub hangers: Vec<Hanger>,
+    pub post_iterate: Option<InsideOutDonut>,
```

Comment:
It looks like you'll only ever have at most one PostIterate op. In that case, easiest would be to make this

```suggestion
    pub post_iterate: Option<Box<dyn PostIterate>>,
```

Then you wouldn't have to make everything generic over `Op: PostIterate` or something

Reply:
I guess I have to live with this killing Debug and Clone derivations?

Reply:
Thing is, I pass this struct around, and Clone a bunch of times.